data/pid_data.py

```diff
@@ -2699,6 +2699,22 @@
        'value': 36864},
     ],
   },
+  # SWISSON AG
+  {'id': 0x5377,
+   'name': 'SWISSON AG',
+   'pids': [
+            # SWPID_AUTO_UNIVERSE_NUMBER
+            {'name': 'SWPID_AUTO_UNIVERSE_NUMBER',
+             'get_request': {'items': []},
+             'get_response': {'items': [{'name': 'universe_number', 'type': 'uint16'}]},
+             'get_sub_device_range': 0,
+             'set_request': {'items': [{'name': 'universe_number', 'type': 'uint16'}]},
+             'set_response': {'items': []},
+             'set_sub_device_range': 0,
+             'value': 0x8082,
```

Comment:
Does this mean you've got a load of other PIDs too, or are they private?

Reply:
No , we haven't got any other private PIDs, just tried to comply with the specification:

> Manufacturer-Specific PID values should be selected by choosing the appropriate category fromTable A-3 and adding an offset of 0x8000 to preserve a logical organization. 

E1-20 6.2.10.2

In our case we used PID DEVICE_LABEL + 0x8000.

Reply:
Well I'm glad someone follows that bit of the spec, I think you might be one of the only ones, even we don't do that! :laughing: